src/Lykke.Service.PayInternal.AzureRepositories/MerchantGroup/MerchantGroupEntity.cs

```diff
@@ -29,27 +30,34 @@ public MerchantGroupEntity(string partitionKey, string rowKey)
             PartitionKey = partitionKey;
             RowKey = rowKey;
         }
+        public static string GenerateRowKey()
+        {
+            return Guid.NewGuid().ToString();
+        }
+        public static string GeneratePartitionKey(string ownerId)
+        {
+            return ownerId;
+        }
         public static class ByOwner
```

Comment:
You should name it by partition, e. g. it should be ByGroup. If it is index better name it IndexBy...